### DISCUSSION TITLE
Camoufox Discussion

### DISCUSSION THREAD

**Discussion by daijro** [daijro/camoufox#11] [Announcements]
## 👋 Welcome!

I've opened discussions as a way to share ideas about Camoufox's development, issues, and potential leaks. :smiley:

**Comment by tserebour** [comment]
When you open the camoufox (excellent work by the way) launcher manually in the terminal,  is there a way to connect to it with playwright like CDP does?
Like get websocket endpoints and connect later? I know this is a CDP thing but i was just wondering.

**Reply by daijro** [reply]
I don't believe so. Juggler (Playwright's protocol for Firefox) works by establishing a pipe between the Playwright core server and Firefox to communicate. This means that Playwright and Firefox (or a fork like Camoufox) must be launched together using Playwright's API to control the browser instance directly over the pipe.

Playwright's can't retroactively attach to a manually launched instance in the same way that CDP does for Chromium-based browsers. But, having the ability to remotely host Camoufox and access it through a WSS endpoint is something I have planned.

**Comment by roamproxy** [comment]
Hi @daijro — sorry to drop this in the general thread (my account can't open a new discussion). Quick README question rather than a dev/leak topic:

Is the **Proxy providers** section in the README open to new providers, and what's the arrangement — paid placement, affiliate, or editorial? I run partnerships at Roam (roamproxy.com); we maintain a Camoufox setup guide already (https://roamproxy.com/use-cases/camoufox-proxy-setup) and can offer a discount code + revenue share, structure negotiable. I emailed admin@cloverlabs.ai twice (Aug 8, Sep 1) but it may have been spam-filtered.

If it's not open or we're too small, a one-liner is all I need and I'll drop it. Thanks for Camoufox either way.

**Reply by daijro** [reply]
Hello, you can contact me at daijro.dev@gmail.com 👍
Cloverlabs isn't actively maintaining the project at the moment.

**Comment by roamproxy** [comment]
Thanks @daijro, appreciate the quick reply — just emailed you at daijro.dev@gmail.com with the details (README proxy-providers slot, a CAMOUFOX15 code, and a recurring rev-share we can align to the other listed providers). No rush, and totally fine if it's not a fit. Thanks again for Camoufox.
